### PR TITLE
Ensure `ember-data` modules polyfill does not apply to any `@ember-data` packages.

### PR DESCRIPTION
**Pull request by runspired**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -511,8 +511,8 @@ module.exports = {
     let addonOptions = config['ember-cli-babel'] || {};
 
     if (addonOptions.disableEmberDataPackagesPolyfill) { return; }
-    // Don't convert ember-data itself!
-    if (this.parent.name === 'ember-data') { return; }
+    // Don't convert ember-data itself or any @ember-data packages!
+    if (this.parent.name === 'ember-data' || this.parent.name.indexOf('@ember-data/') === 0) { return; }
```
Comment (rwjblue) [inline index.js]: Could use `.startsWith` I think (instead of .indexOf() === 0), what do you think?

Comment (runspired) [reply index.js]: @rwjblue I had to add a check that name was actually present. RE startsWith is it compatible with node 6?

Comment (runspired) [reply index.js]: @rwjblue do you think we need to check `parent.name()` as well?

Comment (runspired) [reply index.js]: (I'm guessing no since our own tests would always have a version >= 3.12)

Comment (runspired) [reply index.js]: looks like startsWith has support in node 4 so will use

Comment (rwjblue) [reply index.js]: .name as a function is only going to match the project itself (which means inside ember-data while locally developing), but ember-data can just directly disable the flag in its Babel config. 

Totally up to you if you want to support both, but I don’t think it’s strictly required

Comment (runspired) [reply index.js]: Let’s merge and move on then :)

**Pull request merged**